### ISSUE TITLE
References of kind Qualify are missing method name

### ISSUE THREAD

**Issue by jnyrup** [dotnet/roslyn#34954]
**Version Used**: 
16.1.0 Preview 1.0

**Steps to Reproduce**:
```c#
public class Class1
    {
        public async void Bar() => await Task.Delay(0);

        public async Task Baz() => await Task.Delay(0);
    }
```

1. Find all references on `Task` 
2. Look at the "'Task' references" window.

**Expected Behavior**:
The two references of kind Qualify should both show `Task.Delay(0)`

**Actual Behavior**:
For `Bar` it only shows `Task.(0)`
![image](https://user-images.githubusercontent.com/919634/56020957-9c9a7300-5d08-11e9-934c-7263182e8445.png)

**Comment by CyrusNajmabadi** [comment]
I'm also seeing this. @JoeRobich Did you work on hte new colors?  Is this fallout from that?

**Comment by JoeRobich** [comment]
This is duplicate of https://github.com/dotnet/roslyn/issues/34307 and should be fixed in 16.1 preview2 by https://github.com/dotnet/roslyn/pull/34315
